dev/bots/test/analyze-test-input/root/packages/flutter/lib/bar.dart

```diff
@@ -16,11 +16,9 @@ class Foo {
   @_debugOnly
   final Map<String, String>? bar = kDebugMode ? null : <String, String>{};
 
-  // dart format off
+  // dart format off, checks the annotation works for multiline expressions
```

Comment:
You may have to put this comment on the line above (looks like dartfmt was applied to the next line..)